src/framework/midi/CMakeLists.txt

```diff
@@ -82,6 +90,7 @@ if (OS_IS_MAC)
 elseif (OS_IS_WIN)
     set(MODULE_LINK ${MODULE_LINK} winmm)
 elseif (OS_IS_LIN)
+    # FIX: why does jack sources compile without a find_library here?
```

Comment:
Perhaps this is because the include dirs of some other package are just `/usr/local/include` and Jack is also installed to that location?